### ISSUE TITLE
Please add a link to the 2.2 specification from the understanding documents

### ISSUE THREAD

**Issue by lauracarlson** [w3c/wcag#3292]
The previous versions of WCAG had a link to the specification [for example 2.0's 1.2.1 ](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html) and  [2.1's 1.2.2](https://web.archive.org/web/20190713065229/https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded). The 2.2 understanding docs currently lack a link, [for example 2.2's 1.2.1 ](https://www.w3.org/WAI/WCAG22/Understanding/audio-only-and-video-only-prerecorded).

Many people will find the understanding doc from search and can't use back button.

**Comment by CBID2** [comment]
Hi @lauracarlson! :) I would like to work on this issue

**Comment by lauracarlson** [comment]
@ CBID2  That's great.

**Comment by CBID2** [comment]
Thanks, @lauracarlson. Can I get assigned to this issue?

**Comment by lauracarlson** [comment]
I think that would be a question for the Chairs? @alastc @rachaelbradley @cwadamsoforacle

**Comment by CBID2** [comment]
> I think that would be a question for the Chairs? @alastc @rachaelbradley @cwadamsoforacle

Ok. Also, is [this](https://www.w3.org/WAI/WCAG21/Understanding/enough-time) the specification link you're referring to? Pardon my questions, I'm new to this project :)

**Comment by lauracarlson** [comment]
That is an one of the 2.2 Understanding Documents. The [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/) is the specification.

**Comment by CBID2** [comment]
> That is an one of the 2.2 Understanding Documents. The [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/) is the specification.

Thanks for clarifying @lauracarlson

**Comment by CBID2** [comment]
Hi @lauracarlson! :) I made the PR. It's #3301! :)

**Comment by patrickhlauke** [comment]
to clarify: to take an example, compare and contrast

* https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html

with the way it was previously

* https://web.archive.org/web/20190629062834/https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html

Notice how in the old version, the SC number/title itself at the top linked back to the relevant SC in the spec. This was done as part of the build scripts/XSLT that prepared all those understanding documents for publication.

The fix here is modifying those build scripts to do the same again.

**Comment by patrickhlauke** [comment]
Closing this as a duplicate of https://github.com/w3c/wcag/issues/3289
